psi-ms.obo

```diff
@@ -23617,6 +23617,208 @@ relationship: has_metric_category MS:4000008 ! ID based metric
+is_a: MS:4000003 ! single value
+is_a: MS:4000008 ! ID based
+is_a: QC:4000001 ! QC metric
+
+
+[Term]
+id: MS:4000159
+name: median of TIC values in the shortest RT range in which half of the peptides are identified
+def: "Median of TIC values in the shortest RT range in which half of the peptides are identified"  [PSI:MS]
+is_a: MS:4000003 ! single value
+is_a: MS:4000008 ! ID based
+is_a: QC:4000001 ! QC metric
+
+[Term]
+id: MS:4000160
+name: precursor intensity range
+def: "Minimum and maximum precursor intensity recorded." [PSI:MS]
+is_a: MS:4000009 ! ID free
+is_a: QC:4000001 ! QC metric
+is_a: QC:4000004 ! n-tuple
+
+[Term]
+id: MS:4000161
+name: identified precursor intensity distribution Q1, Q2, Q3
+def: "From the distribution of identified precursor intensities, the quartiles Q1, Q2, Q3" [PSI:MS]
+is_a: MS:4000004 ! n-tuple
+relationship: has_metric_category MS:4000009 ! ID free metric
+relationship: has_metric_category MS:4000022 ! MS2 metric
+relationship: has_value_concept STATO:0000291 ! quantile
+relationship: has_value_type xsd:float ! The allowed value-type for this CV term
+relationship: has_units MS:1000043 ! intensity unit
+
+[Term]
+id: MS:4000162
+name: unidentified precursor intensity distribution Q1, Q2, Q3
+def: "From the distribution of unidentified precursor intensities, the quartiles Q1, Q2, Q3" [PSI:MS]
+relationship: has_metric_category MS:4000009 ! ID free metric
+relationship: has_metric_category MS:4000022 ! MS2 metric
+relationship: has_value_concept STATO:0000291 ! quantile
+relationship: has_value_type xsd:float ! The allowed value-type for this CV term
+relationship: has_units MS:1000043 ! intensity unit
+
+[Term]
+id: MS:4000163
+name: identified precursor intensity distribution mean
+def: "From the distribution of identified precursor intensities, the mean. The intensity distribution of the identified precursors informs about the dynamic range of the acquisition in relation to identifiability." [PSI:MS]
+is_a: MS:4000003 ! single value
+relationship: has_metric_category MS:4000009 ! ID free metric
+relationship: has_value_concept STATO:0000401 ! sample mean
+relationship: has_value_type xsd:float ! The allowed value-type for this CV term
+relationship: has_units MS:1000043 ! intensity unit
+
+[Term]
+id: MS:4000164
+name: unidentified precursor intensity distribution mean
+def: "From the distribution of unidentified precursor intensities, the mean. The intensity distribution of the unidentified precursors informs about the dynamic range of the acquisition in relation to identifiability." [PSI:MS]
+is_a: MS:4000003 ! single value
+relationship: has_metric_category MS:4000009 ! ID free metric
+relationship: has_value_concept STATO:0000401 ! sample mean
+relationship: has_value_type xsd:float ! The allowed value-type for this CV term
+relationship: has_units MS:1000043 ! intensity unit
+
+[Term]
+id: MS:4000165
+name: charged peptides ratio 1+ over 2+
```

Comment:
I think the following are all "IS-3A" related? Could you please try and consolidate with **MS:4000063**?